db/migrate/20210119082215_add_nickname_to_users.rb

```diff
@@ -0,0 +1,6 @@
+class AddNicknameToUsers < ActiveRecord::Migration[5.2]
+  def change
+    add_column :users, :nickname, :string, null: true, after: :email
+    add_index :users, :nickname, unique: true, where: 'nickname IS NOT NULL'
```

Comment:
i think we should leave after: :email

Reply:
Can you explain for about :string type? Is the add_cloumn I do wrong format?

Reply:
ah, sorry, i didnt check line before